src/com/magento/idea/magento2plugin/actions/generation/dialog/validator/NewViewModelValidator.java

```diff
@@ -25,11 +27,12 @@ public static NewViewModelValidator getInstance(NewViewModelDialog dialog) {
 
     public NewViewModelValidator() {
         this.validatorBundle = new ValidatorBundle();
+        this.commonBundle = new CommonBundle();
     }
 
     public boolean validate()
     {
-        String errorTitle = "Error";
+        String errorTitle =commonBundle.message("common.error");
```

Comment:
@eduard13 please, add a whitespace after the `=`

Reply:
@coderimus nice catch. Fixed this one.
Thank you.